test/reporter-html.js

```diff
@@ -100,27 +68,60 @@ QUnit.module( "timing", {
 
 QUnit.test( "setup", function( assert ) {
 	assert.expect( 0 );
-	delayNextSetup = true;
+	this.delayNextSetup = true;
 });
 
 QUnit.test( "basics", function( assert ) {
-	assert.expect( 2 );
-	var previous = getPreviousTests( /^setup$/, /^timing$/ )[ 0 ],
-		runtime = previous.lastChild.previousSibling;
-	assert.ok( /(^| )runtime( |$)/.test( runtime.className ), "Runtime element exists" );
+	assert.expect( 1 );
+	var previous = this.getPreviousTest( assert ),
+		runtime = this.filterClass( previous.getElementsByTagName( "span" ) );
+
 	assert.ok( /^\d+ ms$/.test( runtime.innerHTML ), "Runtime reported in ms" );
 });
 
 QUnit.test( "values", function( assert ) {
 	assert.expect( 2 );
-	var basics = getPreviousTests( /^setup$/, /^timing$/ )[ 0 ],
-		slow = getPreviousTests( /^basics$/, /^timing$/ )[ 0 ];
-	assert.ok( parseInt( basics.lastChild.previousSibling.innerHTML, 10 ) < 100,
+	var basics = this.getPreviousTest( assert ),
+		setup = basics.previousSibling;
+
+	basics = this.filterClass( basics.getElementsByTagName( "span" ) );
+	setup = this.filterClass( setup.getElementsByTagName( "span" ) );
+
+	assert.ok( parseInt( basics.innerHTML, 10 ) < 100,
 		"Fast runtime for trivial test"
 	);
-	assert.ok( parseInt( slow.lastChild.previousSibling.innerHTML, 10 ) > 100,
+	assert.ok( parseInt( setup.innerHTML, 10 ) > 100,
 		"Runtime includes beforeEach"
 	);
 });
 
-})();
+QUnit.module( "source" );
+
+QUnit.test( "setup", function( assert ) {
+	assert.expect( 0 );
```

Comment:
same setup test on the [timing module](https://github.com/jquery/qunit/pull/758/files#r25956696) 

This test block is required in the next block.